tests/fips/openjdk/openjdk_fips.pm

```diff
@@ -0,0 +1,70 @@
+# SUSE's openjdk fips tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: openjdk expect
+#
+# Summary: FIPS: openjdk
+#          Jira feature: SLE-21206
+#          FIPS 140-3: make OpenJDK be able to use the NSS certified crypto
+#
+# Maintainer: Yutao Wang <yuwang@suse.com>
```

Comment:
Please provide the ticket number.

Reply:
Please use this format for ticket.
`# Tags: poo#39071, poo#105591, poo#105999 ....etc.`